docs/docs/porting-to-gatsby.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Porting to Gatsby
+---
+
```

Comment:
This could go in another PR, but it might be nice to add some text here with a call for contributions for other porting docs. Are there other candidates you can think of (we could create some stubs/issues)?

Reply:
I'm not super familiar with many other tools like Create React App but I found this page while searching: https://github.com/tuchk4/awesome-create-react-app#alternatives

Not sure how popular those alternatives are, seems like maybe Next or Razzle could be beneficial though I don't have any experience with either.

Reply:
@gillkyle I think the most common would just be a hand-rolled webpack config. So it's effectively create-react-app but with an opinionated webpack.config.js.

Could have some useful stuff, I can think of a few:

1. moving providers (like ApolloProvider, ThemeProvider, etc.) to gatsby-browser.js
1. Using a webpack alias to alias src/ directory (this is quite common with hand-rolled webpack configs)
1. Breaking apart an App component into Layout components, page components, etc. (you hinted at this earlier with one of your comments!)

So yeah, it's more like "Porting an Existing React App" vs. "Porting Create React App to Gatsby" but perhaps it's pieces of that document instead of a whole document?